teamsupport/config.py

```diff
@@ -1,5 +1,5 @@
-ORG_ID = None
-AUTH_KEY = None
+ORG_ID = 1114960
+AUTH_KEY = '2da0d579-1507-4d81-9112-a3e47f62d636'
```

Comment:
This is public repo so we should probably get a new auth key from team support since this one has been exposed.